optics/tests/Optics/Tests/Labels/Generic.hs

```diff
@@ -152,7 +157,11 @@ howManyGoldFish :: Int
 howManyGoldFish = lengthOf (#pets % folded % #_GoldFish) humanWithFish
 
 hasLazyPets :: Bool
+#if __GLASGOW_HASKELL__ >= 906
+hasLazyPets = orOf (#pets % folded % #"?lazy") human
```

Comment:
why only with GHC-9.6? That's unclear.

Maybe if there were a changelog explaining the addition, this would be clearer too.

Reply:
This requires unrestricted overloaded labels which are in >=9.6

Reply:
Even the `#"..."` syntax. ugh :(